Wikipedia/Code/SectionEditorViewController.swift

```diff
@@ -136,6 +147,31 @@ class SectionEditorViewController: ViewController {
         navigationController?.setNavigationBarHidden(false, animated: false)
     }
     
+    private func removeScriptMessageHandlers() {
+        
+        guard let userContentController = webView?.configuration.userContentController,
+            !scriptMessageHandlers.isEmpty else {
```

Comment:
same as below, not sure if we need this check - but not a problem to keep it

Reply:
Fixed.